client-v2/src/util/storeMiddleware.js

```diff
@@ -40,9 +40,9 @@ const updateStateFromUrlChange: Middleware<State, Action> = ({
   return result;
 };
 
-type PersistCollectionMeta = {|
+type PersistMeta = {|
   // The resource to persist the data to
-  persistTo: 'collection',
+  persistTo: 'collection' | 'clipboard',
```

Comment:
@jonathonherbert this is where I fixed that issue I spoke IRL with you about, let me know your thoughts.

Reply:
Huh, that's interesting - I would have thought the two types would be basically the same thing. I'd like to know why that's not the case! Well done for solving.

Reply:
Me too ...